deposit/src/test/java/edu/unc/lib/deposit/normalize/BagIt2N3BagJobTest.java

```diff
@@ -136,6 +142,12 @@ public URI answer(InvocationOnMock invocation) throws URISyntaxException {
 			children.put(file.getProperty(dprop(model, label)).getString(), file);
 		}
 		
+		ArgumentCaptor<String> arg = ArgumentCaptor.forClass(String.class);
+		verify(depositStatusFactory, times(2)).addManifest(anyString(), arg.capture());
+		List<String> capturedFilenames = Arrays.asList("bagit.txt", "manifest-md5.txt");
```

Comment:
This isn't actually testing the captured filenames, its testing against a hand built list of filenames.  capturedFilenames should be asking the ArgumentCaptor for all of the captured values.  You can also compare two arrayLists to each other with .equals